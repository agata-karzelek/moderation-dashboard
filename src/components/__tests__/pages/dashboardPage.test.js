import React from "react";
import { shallow } from "enzyme";
import DashboardPage from "../../../pages/dashboard";

describe("Test Case For Dashboard", () => {
  test("SnapShot", () => {
    const wrapper = shallow(<DashboardPage />);
    expect(wrapper).toMatchSnapshot();
  });

  test("should render router", () => {
    const wrapper = shallow(<DashboardPage />);
    const element = wrapper.find("Header");
    expect(element).toHaveLength(1);
    expect(element.text()).toEqual("<Header />");
    const element2 = wrapper.find("AppsListing");
    expect(element2).toHaveLength(1);
    expect(element2.text()).toEqual("<AppsListing />");
  });
});
