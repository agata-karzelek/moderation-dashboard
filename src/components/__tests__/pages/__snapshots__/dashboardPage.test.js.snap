// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Case For Dashboard SnapShot 1`] = `
<Fragment>
  <WithStyles(CssBaseline) />
  <Header />
  <AppsListing />
</Fragment>
`;
